Add  payload size limitation for CDI (#6265)
braze-inc/braze-docs: _docs/_user_guide/data_and_analytics/cloud_ingestion/overview.md

## Product limitations

| Limitations            | Description                                                                                                                                                                        |
| ---------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Number of integrations | There is no limit on how many integrations you can set up. However, you will only be able to set up one integration per table or view.                                             |
| Number of rows         | There is no limit on the number of rows you can sync. Each row will only be synced once, based on the `UPDATED` column.                                                            |

> | Attributes per row     | Each row should contain a single user ID and a JSON object with up to 50 attributes. Each key in the JSON object counts as one attribute (i.e., an array counts as one attribute). |

Reviewer: ```suggestion
| Attributes per row     | Each row should contain a single user ID and a JSON object with up to 50 attributes. Each key in the JSON object counts as one attribute (that is, an array counts as one attribute). |
```
